<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieMultiScrapeDialog</class>
 <widget class="QDialog" name="MovieMultiScrapeDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>556</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Movie Multi Scrape</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../data/MediaElch.qrc">
    <normaloff>:/img/MediaElch.png</normaloff>:/img/MediaElch.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Please select the scraper and the infos you want to be loaded. MediaElch will use the best result for each movie you selected.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Scraper</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboScraper"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Infos to load</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="MyCheckBox" name="chkSet">
          <property name="text">
           <string>Set</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="MyCheckBox" name="chkTitle">
          <property name="text">
           <string>Title</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="MyCheckBox" name="chkRating">
          <property name="text">
           <string>Rating</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="MyCheckBox" name="chkOverview">
          <property name="text">
           <string>Overview</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="MyCheckBox" name="chkPoster">
          <property name="text">
           <string>Poster</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="MyCheckBox" name="chkCountries">
          <property name="text">
           <string>Countries</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="MyCheckBox" name="chkRuntime">
          <property name="text">
           <string>Runtime</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="MyCheckBox" name="chkActors">
          <property name="text">
           <string>Actors</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="MyCheckBox" name="chkCertification">
          <property name="text">
           <string>Certification</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="MyCheckBox" name="chkTrailer">
          <property name="text">
           <string>Trailer</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="MyCheckBox" name="chkTagline">
          <property name="text">
           <string>Tagline</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="MyCheckBox" name="chkReleased">
          <property name="text">
           <string>Released</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="MyCheckBox" name="chkWriter">
          <property name="text">
           <string>Writer</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="MyCheckBox" name="chkDirector">
          <property name="text">
           <string>Director</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="MyCheckBox" name="chkStudios">
          <property name="text">
           <string>Studios</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="MyCheckBox" name="chkGenres">
          <property name="text">
           <string>Genres</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="MyCheckBox" name="chkBackdrop">
          <property name="text">
           <string>Backdrop</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="MyCheckBox" name="chkThumb">
          <property name="text">
           <string>Thumb</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="MyCheckBox" name="chkBanner">
          <property name="text">
           <string>Banner</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="MyCheckBox" name="chkCdArt">
          <property name="text">
           <string>CD Art</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="MyCheckBox" name="chkClearArt">
          <property name="text">
           <string>Clear Art</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="MyCheckBox" name="chkLogo">
          <property name="text">
           <string>Logo</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="MyCheckBox" name="chkTags">
          <property name="text">
           <string>Tags</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkUnCheckAll">
        <property name="text">
         <string>(Un)Check all</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="chkAutoSave">
     <property name="text">
      <string>Automatically save each movie after scraping</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkOnlyImdb">
     <property name="text">
      <string>Update only movies with IMDb ID/TheMovieDb Id</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="movieCounter">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>1/20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="movie">
       <property name="text">
        <string>Movie</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressMovie">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressAll">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartScraping">
       <property name="text">
        <string>Start Scraping</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">ui/small_widgets/MyCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../data/MediaElch.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>MovieMultiScrapeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>MovieMultiScrapeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
